incubator/xray/templates/rabbitmq-ha-configmap.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-rabbitmq-ha
+  labels:
+    app: {{ template "xray.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
```

Comment:
The current `_helpers.tpl` you get with `helm create` also adds a `chart` partial. I'd suggest you update accordingly.